config/routes.rb

```diff
@@ -0,0 +1,18 @@
+Rails.application.routes.draw do
+  devise_for :users
+  
+  resources :recipes, except: [:edit] do
+  resources :recipe_foods, only: [:new, :create, :destroy, :edit, :update]
+  end
+  
+  resources :foods
+  resources :users
+  
```

Comment:
- [ ] Kindly only leave out routes that are being used, for instance, PUT/patch is not used anywhere.

Reply:
Great review @elisha2kyakpo1 👏 👏 

We will work to fix the required changes 💯